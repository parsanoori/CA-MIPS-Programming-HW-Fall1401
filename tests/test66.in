57
13950081
183805679
131917993
52221614
754210208
1042741412
884769523
376498702
200980821
165056509
649808654
191346951
25189860
335060084
506331020
245286939
76059419
97060815
42199177
209145134
275062957
311797458
706488410
184258552
499144927
593919884
1068389340
937682963
739273732
671937666
404957413
163532599
530072708
469085854
637039736
283342870
173940473
618552786
474066232
284224338
121454985
365405652
682012521
113768164
14845306
241508232
941196400
143835743
711938647
278188265
88872069
485793938
618843659
220131777
998517294
472084411
242487783