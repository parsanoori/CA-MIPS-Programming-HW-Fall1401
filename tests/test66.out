13950081
14845306
25189860
42199177
52221614
76059419
88872069
97060815
113768164
121454985
131917993
143835743
163532599
165056509
173940473
183805679
184258552
191346951
200980821
209145134
220131777
241508232
242487783
245286939
275062957
278188265
283342870
284224338
311797458
335060084
365405652
376498702
404957413
469085854
472084411
474066232
485793938
499144927
506331020
530072708
593919884
618552786
618843659
637039736
649808654
671937666
682012521
706488410
711938647
739273732
754210208
884769523
937682963
941196400
998517294
1042741412
1068389340